src/ImageRunModal.jsx

```diff
@@ -90,12 +90,12 @@ const EnvVar = ({ id, item, onChange, idx, removeitem, additem, itemCount }) =>
             <FormGroup className="pf-m-6-col-on-md" label={_("Key")} fieldId={id + "-key-address"}>
                 <TextInput id={id + "-key"}
                        value={item.envKey || ''}
-                       onChange={value => handleEnvValue('envKey', value, idx, onChange, additem, itemCount, item.envValue)} />
+                       onChange={(_, value) => handleEnvValue('envKey', value, idx, onChange, additem, itemCount, item.envValue)} />
             </FormGroup>
             <FormGroup className="pf-m-6-col-on-md" label={_("Value")} fieldId={id + "-value-address"}>
                 <TextInput id={id + "-value"}
                        value={item.envValue || ''}
-                       onChange={value => handleEnvValue('envValue', value, idx, onChange, additem, itemCount, item.envKey)} />
+                       onChange={(_, value) => handleEnvValue('envValue', value, idx, onChange, additem, itemCount, item.envKey)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230606-121542-0ae9eb51-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)